crates/spawner/src/spawner.rs

```diff
@@ -72,6 +73,7 @@ fn spawn(
         match object_type {
             ObjectType::Active(active_type) => {
                 entity_commands.insert(Active);
+                entity_commands.insert(Battery::default()); // a bit of a placeholder as final solution is likely going to be in `SolidObject` configured though the object config
```

Comment:
TL;DR: I would remove the comment.

Very likely the implementation will look like you suggest. However, we should keep our plans and ideas in GitHub issues & discussions. Plans tend to change over time and it is hard to keep all such comments up-to-date, well organized and searchable (e.g. via labels).